operations/observability/Observability.md

```diff
@@ -0,0 +1,336 @@
+# Install and Upgrade Observability Framework
+
+## Overview
+
+The Install and Upgrade Observability Framework collects Goss test health check results and presents them in a visual Grafana dashboard, providing key insights into the health of the system.
+
+The automated Time to Install (TTI) and Time to Upgrade (TTU) provide the time and other important details for each portion of the HPC install, such as when and on what machine a given command was run.
+
+For install of each product stream, the observability framework records the time spent in actual work, debugging, and waiting, as well as the total duration.
+
+The following is a list of the most important features of the framework:
+
+- Provides a consistent framework for health checks/validation, metrics, and reporting for all product stream  installs and upgrades.
+  - Inside-out views
+    - Timing dashboard view that focuses on a single operation, and from that operation an administrator can zoom out to view the total product installation details.
+    - Goss test dashboard view that focuses on a single test, and from that test an administrator can zoom out to view the total product test details.
+  - Outside-in views
+    - Timing dashboard view that shows total product details, and from that view an administrator can zoom in to focus on details of a single installation operation.
+    - Goss test dashboard view that shows total product test details, and from that view an administrator can zoom in to focus on details of a single test.
+  - Product stream health
```

Comment:
Seems like this should have a one-line description like the other features above do.